cmd/bb_scheduler/build_queue_state_service.go

```diff
@@ -250,6 +294,102 @@ var (
     </table>
   </body>
 </html>
+`))
+	listActiveInvocationsTemplate = template.Must(template.New("ListActiveInvocations").Funcs(templateFuncMap).Parse(`
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Active bazel invocations</title>
+    <style>
+      html { font-family: sans-serif; }
+      table { border-collapse: collapse; }
+      table, td, th { border: 1px solid black; }
+      td, th { padding-left: 5px; padding-right: 5px; }
+      .text-monospace { font-family: monospace; }
+    </style>
+  </head>
+  <body>
+    {{$instanceName := .InstanceName}}
+    <h1>Recent invocations</h1>
+    <p>Instance name: {{$instanceName}}</p>
+    <p><a href="inactive_invocations?instance_name={{.InstanceName}}">Show inactive invocations</a></p>
+    <table>
+      <thead>
+        <tr>
+          <th>Invocation<br/>ID</th>
+          <th>First<br/>operation</th>
+          <th>Priority</th>
+          <th>Queued<br/>operations</th>
+          <th>Executing<br/>operations</th>
+          <th>Finished<br/>recent<br/>operations</th>
+          <th>Keywords</th>
+          <th>Control</th>
+          <th>Timeout</br>
+        </tr>
+      </thead>
+      {{$browserURL := .BrowserURL}}
+      {{$now := .Now}}
+      {{range .Invocations}}
+        <tr>
+          <td><a href="operations?invocation_id={{.InvocationID}}">{{abbreviate .InvocationID}}</a></td>
+          <td>{{to_timestamp .InvocationTimestamp}}</td>
+          <td>{{.Priority}}</td>
+          <td><a href="operations?invocation_id={{.InvocationID}}&selection=Queued">{{.QueuedOperationsCount}}</a></td>
+          <td><a href="operations?invocation_id={{.InvocationID}}&selection=Executing">{{.ExecutingOperationsCount}}</a></td>
+          <td><a href="operations?invocation_id={{.InvocationID}}&selection=Finished">{{.FinishedOperationsCount}}</a></td>
+          <td>{{list_keywords .Keywords}}</td>
+          <td> <button {{potential_disable .Canceled}} onclick="window.location.href='kill_invocation?name={{.InvocationID}}';" value="">Cancel</button></td>
```

Comment:
The same holds for potential_disable: `{{if .Canceled}}disabled{{end}}` seems to be as long as calling into this helper function.